addons/mail/static/src/components/attachment/attachment.js

```diff
@@ -94,10 +94,10 @@ class Attachment extends Component {
         if (this.detailsMode === 'card') {
             size = '38x38';
         } else {
-            size = '160x160';
+            size = '200x200';
```

Comment:
This should actually depend on `props.imageSize`.

And I would add a comment to say this should be sync with `.o_Attachment_image` `width` and `height`.
And also a comment on the css file to say it should be sync with the value here.

Reply:
@seb-odoo .. you mean we have to make it separate based on props.imageSize and comments on the files? just review the changes.